core/templates/dev/head/domain/collection/CollectionRightsBackendApiService.js

```diff
@@ -48,6 +66,15 @@ oppia.factory('CollectionRightsBackendApiService', [
 
       return {
         /**
+         * Gets a collection's rights, given its ID.
+         */
+        getCollectionRights: function(collectionId) {
```

Comment:
We use fetchCollectionRights for getting it from the datastore, and getCollectionRights for getting the local copy (see e.g. domain/collection/EditableCollectionBackendApiService.js). I think here you mean fetch instead of get?